job/common/src/main/java/alluxio/job/util/SerializationUtils.java

```diff
@@ -111,4 +116,28 @@ public static Serializable deserialize(byte[] bytes, String errorMessage) {
       Map<S, Collection<T>> map) {
     return Maps.transformValues(map, ArrayList::new);
   }
+
+  /**
+   * Parse the actual JSON result from the benchmark output. Output might contain interference logs
+   * (for example, some warn level logs indicating that the listing file time is too long).
+   * @param result the output of benchmark
+   * @return the actual result in JSON format
+   */
+  public static String parseBenchmarkResult(String result) {
+    String[] taskResults = result.split("\n");
+    boolean isActualResultStart = false;
+    StringBuilder actualResult = new StringBuilder();
+
+    for (int i = 0; i < taskResults.length; i++) {
+      if (isActualResultStart) {
+        actualResult.append(taskResults[i]);
+      }
+      if (taskResults[i].startsWith(BENCHMARK_RESULT_TAG)) {
+        isActualResultStart = true;
+        // We need to take account of the cluster mode.
+        actualResult = new StringBuilder();
```

Comment:
This is a bit special, because cluster mode will print the tag twice, but we only need the content after the last tag.